tests/libs/datasets/timeseries_test.py

```diff
@@ -1612,3 +1612,28 @@ def test_write_read_dataset_pointer_with_source_url(tmpdir):
     assert source_url_read[CommonFields.ICU_BEDS] == [url_str1]
     # Copy to a set because the order of the URLs in the source_url may change.
     assert set(source_url_read[CommonFields.CASES]) == {url_str2, url_str3}
+
+
+@pytest.mark.parametrize("last_value,is_outlier", [(0.02, False), (0.045, True)])
+def test_remove_test_positivity_outliers(last_value, is_outlier):
+    values = [0.015] * 7 + [last_value]
+    dataset = test_helpers.build_default_region_dataset({CommonFields.TEST_POSITIVITY_7D: values})
+    dataset = timeseries.drop_tail_positivity_outliers(dataset)
```

Comment:
call these dataset_in and dataset_out to avoid reusing the name. Then in the else block `test_helpers.assert_dataset_like(dataset_in, dataset_out)`